Project.toml

```diff
@@ -5,6 +5,7 @@ version = "0.31.4"
 
 [deps]
 Combinatorics = "861a8166-3701-5b0c-9a16-15d98fcdc6aa"
+DataAPI = "9a962f9c-6df0-11e9-0e5d-c546b8b5ee8a"
```

Comment:
We can get the `describe` name directly from StatsBase.jl, which is a better dependency than DataAPI.jl in this case.